tests/Fixer/Strict/DeclareStrictTypesFixerTest.php

```diff
@@ -141,11 +155,11 @@ public function provideMessyWhitespacesCases()
     {
         return [
             [
-                "<?php declare(strict_types=1);\r\nphpinfo();",
+                "<?php\r\ndeclare(strict_types=1);\r\n\tphpinfo();",
```

Comment:
> If we want to match (proposed) PSR-12 by default, we need the add the newline by default.

who told you that? you must not change default behaviour of fixer, it will fail on any integration usage that are using this rule.
if you want to change behaviour of fixer, the only way is to introduce option for that leaving default behaviour the same